### ISSUE TITLE
Fresh shopify cli project not working in yarn workspaces

### ISSUE THREAD

**Issue by MoSattler** [Shopify/shopify-app-template-remix#600]
Yarn v1.22.21

Steps to reproduce:
```bash
mkdir testrepo && cd testrepo
```
```bash
echo '{
  "name": "testrepo",
  "private": true,
  "version": "1.0.0",
  "workspaces": [
    "testapp"
  ]
}' > package.json
```
```bash
yarn create @shopify/app --package-manager=yarn --name=testapp --template=remix --flavor=typescript
```

Once installation is done, add arbitrary `version` field to `package.json` (required for workspaces)

```bash
yarn workspace testapp run dev
```

When asked `Create this project as a new app on Shopify?` choose `No, connect it to an existing app` and select an app.

The output then is:
```
 error ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                                                             │
│  Fix a schema error in shopify.app.toml:                                                                                    │
│  [                                                                                                                          │
│    {                                                                                                                        │
│      "code": "invalid_type",                                                                                                │
│      "expected": "string",                                                                                                  │
│      "received": "undefined",                                                                                               │
│      "path": [                                                                                                              │
│        "name"                                                                                                               │
│      ],                                                                                                                     │
│      "message": "Required"                                                                                                  │
│    },                                                                                                                       │
│    {                                                                                                                        │
│      "code": "invalid_type",                                                                                                │
│      "expected": "object",                                                                                                  │
│      "received": "undefined",                                                                                               │
│      "path": [                                                                                                              │
│        "auth"                                                                                                               │
│      ],                                                                                                                     │
│      "message": "Required"                                                                                                  │
│    },                                                                                                                       │
│    {                                                                                                                        │
│      "code": "invalid_type",                                                                                                │
│      "expected": "object",                                                                                                  │
│      "received": "undefined",                                                                                               │
│      "path": [                                                                                                              │
│        "webhooks"                                                                                                           │
│      ],                                                                                                                     │
│      "message": "Required"                                                                                                  │
│    },                                                                                                                       │
│    {                                                                                                                        │
│      "code": "invalid_type",                                                                                                │
│      "expected": "string",                                                                                                  │
│      "received": "undefined",                                                                                               │
│      "path": [                                                                                                              │
│        "application_url"                                                                                                    │
│      ],                                                                                                                     │
│      "message": "Required"                                                                                                  │
│    },                                                                                                                       │
│    {                                                                                                                        │
│      "code": "invalid_type",                                                                                                │
│      "expected": "boolean",                                                                                                 │
│      "received": "undefined",                                                                                               │
│      "path": [                                                                                                              │
│        "embedded"                                                                                                           │
│      ],                                                                                                                     │
│      "message": "Required"                                                                                                  │
│    }                                                                                                                        │
│  ]   
```

**Comment by MoSattler** [comment]
Turns out it's mot working with yarn in general, will create new issue
